userscripts/Custom Race Presets (Torn PDA).js

```diff
@@ -77,6 +76,10 @@ function fillPreset(index) {
 		$('#select-allow-upgrades').selectmenu();
 		$('#select-allow-upgrades-menu > li:contains(' + race.upgradesAllowedString + ')').mouseup();
 	}
+	if ("waitTime" in race) {
+		$('#wait-time').selectmenu();
+		$('#wait-time-menu > li:contains(' + race.waitTime + ')').mouseup();
```

Comment:
if `race.waitTime` > 1 we should get the utc time string in x minutes, and round it up to the nearest 15 minutes. Will need to handle the dates (check if it's today, tomorrow, or the date beyond that).
If it's not greater than 1, set it to ASAP

Reply:
I'm not much of a racer myself so you're quite possibly right, I'm just avoidant of breaking existing behaviours and presets in the process.

Perhaps we could do both? Handle strings as precise times, and numbers as "in x minutes"? That way existing presets should continue working as expected, and people stewarding can also set specific times.

Reply:
Sure, I'll see what I can do. Though the preset time have been broken for over two years, so I'm not sure anyone will notice. https://www.torn.com/forums.php#/p=threads&f=21&t=16283415&b=0&a=0